sdk/core/src/index.ts

```diff
@@ -590,62 +461,27 @@ var ReactMoE = {
    * @param {JSONObject} pushPayload
    */
   passFcmPushPayload: function (pushPayload: object) {
-    commonValidationCheck();
     MoEngageLogger.verbose("Will process push payload.");
-    let payload = getMoEPushCampaignJson(pushPayload, PUSH_SERVICE_FCM, moeAppId);
-
     if (Platform.OS == PLATFORM_ANDROID) {
-      MoERNAndroid.passFcmPushPayload(payload);
-    } else {
-      MoEngageLogger.debug("This api is not supported on iOS platform.");
+      let payload = getMoEPushCampaignJson(pushPayload, PUSH_SERVICE_FCM, moeAppId);
+      MoEReactBridge.passFcmPushPayload(payload);
```

Comment:
lets's add back the debug log for not-supported

Reply:
@umangmoe done